tests/IpFilterTest.php

```diff
@@ -32,9 +32,10 @@ public function ipNotAllowedDataProvider(): array
             'basic' => [['REMOTE_ADDR' => '8.8.8.8']],
             'does not exist' => [[]],
             'empty string' => [['REMOTE_ADDR' => '']],
+            'invalid IP' => [['REMOTE_ADDR' => '1']],
             'with subnet' => [['REMOTE_ADDR' => '192.168.5.32/11']],
             'with negation' => [['REMOTE_ADDR' => '!192.168.5.32/32']],
```

Comment:
Can `REMOTE_ADDR` be cidr and use negotiation?

Reply:
No. `REMOTE_ADDR` is always IP.